crucible-mir/crucible-mir.cabal

```diff
@@ -115,7 +115,7 @@ library
   import: shared
 
   build-depends: base >= 4.11 && < 5,
-                 aeson < 2.3,
+                 aeson >= 2.0 && < 2.3,
```

Comment:
Reviewers' note: I raised the lower version bounds on `aeson` to `>= 2.0` because `Mir.ParseTranslate` now explicitly requires the `Data.Aeson.KeyMap` module, which was first introduced in `aeson-2.0`.

Reply:
I don't mind this bump, but if you feel it's undesirable, I think you could do the same lookup via [`(.:)`](https://hackage.haskell.org/package/aeson-2.2.3.0/docs/Data-Aeson.html#v:.:) instead.